shared/src/commonMain/kotlin/com/bunbeauty/shared/data/mapper/cart_product/CartProductMapper.kt

```diff
@@ -31,12 +32,28 @@ class CartProductMapper : ICartProductMapper {
                         Category(
                             uuid = menuProductWithCategoryEntity.categoryUuid,
                             name = menuProductWithCategoryEntity.categoryName,
-                            priority = menuProductWithCategoryEntity.priority
+                            priority = menuProductWithCategoryEntity.categoryPriority
                         )
                     },
                     visible = firstCartProductWithCategoryEntity.visible,
-                    isRecommended = firstCartProductWithCategoryEntity.isRecommended
-                )
+                    isRecommended = firstCartProductWithCategoryEntity.isRecommended,
+                    additionGroups = emptyList(),
+                ),
+                cartProductAdditionList = groupedCartProductWithCategoryEntityList.mapNotNull { cartProductWithMenuProductEntityList ->
+                    cartProductWithMenuProductEntityList.cartProductAdditionUuid?.let {
+                        CartProductAddition(
+                            uuid = cartProductWithMenuProductEntityList.cartProductAdditionUuid,
+                            name = cartProductWithMenuProductEntityList.cartProductAdditionName
```

Comment:
`name` should be not nullable 
`fullName` should be nullable

Reply:
need sync